AudioQnA/tests/test_compose_on_rocm.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
```

Comment:
Not sure about the issue, `yaml: line 3: did not find expected key`, can you add `set -ex` to check?